app/views/users/new.html.erb

```diff
@@ -0,0 +1,20 @@
+<% provide(:title ,"Sign up") %>
+<h1><%= t"users.new.signup" %></h1>
+<div class="row">
+  <div class="row-md-6 col-md-offset-3">
+    <%= form_for(@user) do |f| %>
+      <%= render "shared/error_messages" %>
+
+      <%= f.label t"users.new.name" %>
```

Comment:
```suggestion
      <%= f.label :name %>
```
sử dụng i18n để khai báo tên thuộc tính trong model
https://guides.rubyonrails.org/i18n.html#translations-for-active-record-models